run.py

```diff
@@ -477,18 +477,17 @@ def main() -> None:
     )
     parser.add_argument(
         "--metrics-gpu-backend",
-        choices=["dcgm", "default"],
+        choices=["dcgm", "default", "nvml"],
```

Comment:
Same here, let's change the default backend name to `torch`

Reply:
renamed in https://github.com/pytorch/benchmark/pull/2501/commits/8b48eeaecac07f8649241d5d8287b207cccd3c55